.travis.yml

```
@@ -31,11 +31,19 @@ jobs:
     - stage: linting
       before_script: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.23.3
       script: ./scripts/verify.sh
+      after_success:
+        - echo "Linting check succeeded and your code changes passed in this test"
```

Can we avoid using a single element list?
```suggestion
      after_success: echo "Linting check succeeded"
```

The second part of the sentence doesn't add anything.